src/Chart.php

```diff
@@ -28,11 +28,15 @@ class Chart extends View
         ['rgba(75, 192, 192, 0.2)', 'rgba(75, 192, 192, 1)'],
         ['rgba(153, 102, 255, 0.2)', 'rgba(153, 102, 255, 1)'],
         ['rgba(255, 159, 64, 0.2)', 'rgba(255, 159, 64, 1)'],
+        ['rgba(20, 20, 20, 0.2)', 'rgba(20, 20, 20, 1)']
```

Comment:
does this come from some design guide? can this can abstracted for unlimited colors?